src/layouts/shared/page-header.module.css

```diff
@@ -0,0 +1,273 @@
+.pageHeader {
+  background-color: var(--static-background-background_0-background);
+}
+
+.pageHeader__content {
+  width: 100%;
+  max-width: var(--maxPageWidth);
+  margin: 0 auto;
```

Comment:
Looking in Figma I think this should be `margin: 0`

Reply:
You sure? The sketches don't show many examples of wide screens.

Reply:
It looks so in this frame 🤔 

<img width="924" alt="image" src="https://github.com/AtB-AS/planner-web/assets/43166974/4f97b849-d98c-4764-983a-e99e1dd86b11">

Reply:
Let's have a design review meeting at the end of this week to clean up stuff like this. Leaving it as is for now to keep it similar to the webshop's.